tests/device_tests/bitcoin/payment_req.py

```diff
@@ -98,3 +104,72 @@ def make_payment_request(
         nonce=nonce,
         signature=payment_req_signer.sign_digest_deterministic(h_pr.digest()),
     )
+
+
+def make_coinjoin_request(
```

Comment:
Please make this function deterministic so the tests that use the function are deterministic as well.

Reply:
https://github.com/trezor/trezor-firmware/pull/2577/commits/8f3bae4d87c0a0d62b145b3ec8b53bf7a3d9e7d6